build-conf/Transfer.properties

```diff
@@ -20,6 +20,13 @@ transfer_xmlPDS=${hlq}.XML
 #
 # This is using the DBB PropertyMapping syntax allowing multiple mappings of
 #  target source dataset definitions to the required options
-# 
+#
 transfer_dsOptions=cyl space(1,1) lrecl(80) dsorg(PO) recfm(F,B) dsntype(library) :: transfer_srcPDS, transfer_jclPDS
 transfer_dsOptions=cyl space(1,1) lrecl(144) dsorg(PO) recfm(F,B) dsntype(library) :: transfer_xmlPDS
+
```

Comment:
Please remove this setting from `build-conf`.

We should follow the same patterns like the existing language scripts and add a DBB file property pattern in `application-conf/file.properties`, like the below:

```
#
# Override of the defaultcopyMode to transfer the file to the target library
# Available modes are ASA_TEXT, BINARY, LOAD, and TEXT
# Documentation: https://www.ibm.com/docs/api/v1/content/SS6T76_2.0.0/javadoc/com/ibm/dbb/build/DBBConstants.CopyMode.html
# transfer_copyMode=BINARY :: **/*.object
```
```

Reply:
@dennis-behm I've addressed this feedback (and your related [comment](https://github.com/IBM/dbb-zappbuild/pull/428#discussion_r1361149183) for `samples/application-conf/Transfer.properties`) in the latest [commit](https://github.com/IBM/dbb-zappbuild/pull/428/commits/66806e19a293d5b4e266882f8819379a714d24a7).

Please let us know if there are any additional changes you would like to see!